ansible/group_vars/all.yml

```diff
@@ -0,0 +1,5 @@
+---
+
+# Use python3 on the remote servers
+# Ubuntu doesn't ship with python3 preinstalled anymore.
```

Comment:
Speaking as an outsider at least, this comment doesn't really make much sense to me. It looks like we're configuring to use an interpreter that isn't installed in various places? Not that I really know what's going on with this file anyway...

Reply:
Uh, woops, the comment is wrong, it should be "Ubuntu doesn't ship with python2 preinstalled anymore". Ansible by default uses Python 2, but it can be configured to run with Python 3.